src/main/java/core/basesyntax/FigureTwo.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class FigureTwo extends Figure {
+
+    private double firsSide; // длина
+    private double secondSide; // ширина
+
+    public FigureTwo(String color, double firsSide, double secondSide) {
+        super(color);
+        this.firsSide = firsSide;
```

Comment:
There is a typo in the assignment `this.firsSide = firsSide;`. It should be `this.firstSide = firstSide;` to match the corrected variable name.